src/tsickle.ts

```diff
@@ -774,15 +774,21 @@ class Annotator extends ClosureRewriter {
     }
   }
 
+  static readonly FILEOVERVIEW_COMMENTS =
+      new Set(['fileoverview', 'externs', 'modName', 'mods', 'pintomodule']);
```

Comment:
Yeah, only comments containing `@fileoverview` have `@suppress` parsed in them, and of those, only the last. `@mods` is only parsed out of the first (?) `@fileoverview` by some tooling, but other tooling considers all. So we have every flavour, covering all bases ;-)